src/features/linters/ruff.ts

```diff
@@ -58,11 +70,14 @@ export class Ruff extends BaseLinter {
   private fixToWorkspaceEdit(filename: string, fix: IRuffFix): WorkspaceEdit | null {
     if (!fix) return null;
 
+    // TODO: Support multiple edits.
+    const edit = fix.edits[0];
```

Comment:
Not sure what the best way to support this is. But at least this fixes the immediate parsing error.

Reply:
By the way, here's an example of a fix with multiple edits.

```python
# foo.py
import subprocess
subprocess.run([], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

```console
$ ruff --format json --select UP foo.py
[
  {
    "code": "UP022",
    "message": "Sending stdout and stderr to pipe is deprecated, use `capture_output`",
    "fix": {
      "message": "Replace with `capture_output` keyword argument",
      "edits": [
        {
          "content": "capture_output=True",
          "location": {
            "row": 3,
            "column": 19
          },
          "end_location": {
            "row": 3,
            "column": 41
          }
        },
        {
          "content": "",
          "location": {
            "row": 3,
            "column": 41
          },
          "end_location": {
            "row": 3,
            "column": 65
          }
        }
      ]
    },
    "location": {
      "row": 3,
      "column": 1
    },
    "end_location": {
      "row": 3,
      "column": 67
    },
    "filename": "/<redacted>/foo.py",
    "noqa_row": 3
  }
]
```